frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -736,7 +736,7 @@ local VirtualKeyboard = FocusManager:new{
     umlautmode = false,
     layout = {},
 
-    width = Screen:scaleBySize(600),
+    width = nil,
```

Comment:
You can just remove this line (to not document there is such an option), if any provided width is going to be overriden by your `self.width = Screen:getWidth()` in `:init()`